engine/context/aci-integration.md

```diff
@@ -25,12 +25,12 @@ In addition, the integration between Docker and Microsoft developer technologies
 
 To deploy Docker containers on Azure, you must meet the following requirements:
 
-1. Download and install Docker Desktop Edge version 2.3.2.0 or later.
+1. Download and install Docker Desktop Stable version 2.3.0.5 or later, or Edge version 2.3.2.0 or later.
 
-    - [Download for Mac](https://desktop.docker.com/mac/edge/Docker.dmg){: target="_blank" class="_"}
-    - [Download for Windows](https://desktop.docker.com/win/edge/Docker%20Desktop%20Installer.exe){: target="_blank" class="_"}
+    - [Download for Mac](https://desktop.docker.com/mac/stable/Docker.dmg){: target="_blank" class="_"}

```

Comment:
I left these as edge to encourage more people to use edge. Not sure what @nebuk89 would like us to do though

Reply:
Agree, saying it is in both but pointing to edge is good for now :) a short term issue either way